oms/src/main/java/com/gfg/oms/Request.java

```diff
@@ -0,0 +1,19 @@
+package com.gfg.oms;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+import java.io.Serializable;
+
+@Getter
+public class Request implements Serializable {
+    protected final int api = 1;
+    protected final int id = 1;
+
+    @JsonProperty("apikey")
+    protected String apiKey;
+
+    public void setApiKey(String apiKey) {
```

Comment:
Would be better to inject it via constructor.

Reply:
I don't know how to do it. I tried ways like https://github.com/rzwitserloot/lombok/issues/898 but it didn't work. Seems that only way to achieve this is to use `@Builder`. Do you know how to it without introducing builder/factory?

Reply:
Why not using constructor?

Reply:
Can you please point me to documentation on how? Or I can use not lombok annotation but write constructor by my own.
I kind a like current solution because child classes of Request does not care about apiKey and it is injected right before sending request.